executor/join_test.go

```diff
@@ -942,7 +942,8 @@ func (s *testSuite2) TestHashJoin(c *C) {
 	tk.MustExec("insert into t1 values(1,1),(2,2),(3,3),(4,4),(5,5);")
 	tk.MustQuery("select count(*) from t1").Check(testkit.Rows("5"))
 	tk.MustQuery("select count(*) from t2").Check(testkit.Rows("0"))
-	tk.MustExec("set @@tidb_max_chunk_size=1;")
+	tk.MustExec("set @@tidb_init_chunk_size=1;")
+	//tk.MustExec("set @@tidb_max_chunk_size=1;")
```

Comment:
useless line?

Reply:
no, just remind you want to change max but you can not do that.

Reply:
I do not really understand what you mean here.
Can't we leave a comment here?